### PR TITLE
Added tests for removeMember

### PR DESCRIPTION
**Pull request by carterclark**

### CODE REVIEW

File: src/store/facade/GroceryStore.java [hunk 1]
```diff
@@ -479,6 +481,9 @@ public Result addProduct(Request request) {
 		Result result = new Result();
 		String productId = productsList.add(new Product(request.getProductName(), request.getProductId(),
 				request.getProductCurrentPrice(), request.getProductStockOnHand(), request.getProductReorderLevel()));
+
+		result.setProductFields(productsList.searchById(productId));
```
Comment (paveldanek) [inline src/store/facade/GroceryStore.java]: This should be moved lower, inside the if clause (line 488); product fields should only be set if action was successful.

**Review by paveldanek — CHANGES_REQUESTED** [review]
I left a comment directly in your GroceryStore.java reviewed code.

File: src/store/facade/GroceryStore.java (hunk 1 above)
Comment (carterclark) [reply src/store/facade/GroceryStore.java]: Very good point,

**Review by Chris-LB — APPROVED** [review]
Looks good Big C

**Review by paveldanek — APPROVED** [review]

**Pull request merged**